.github/workflows/core.yml

```
@@ -163,13 +177,21 @@ jobs:
         coins: [universal]
         type: [debuglink]
         asan: [noasan]
+        n4w1: [false]
         exclude:
           - type: normal
```

actionlint (1.7.12)

[error] value "normal" in "exclude" does not match in matrix "type" combinations. possible values are "debuglink"

(matrix)